src/Geopilot.Frontend/src/pages/admin/organisations.tsx

```diff
@@ -1,143 +1,113 @@
-        icon: <LinkOffIcon />,
-        action: () => {
-          const organisation = row as unknown as Organisation;
-          organisation.mandates = [];
-          organisation.users = [];
-          saveOrganisation(organisation);
-        },
-        color: "error",
-        variant: "contained",
-      },
-    ]);
-  }
+  const startEditing = (id: GridRowId) => {
+    navigateTo(`/admin/organisations/${id}`);
+  };
 
   useEffect(() => {
     if (user?.isAdmin) {
       if (organisations === undefined) {
         loadOrganisations();
       }
-
-      if (mandates === undefined) {
-        loadMandates();
-      }
-
-      if (users === undefined) {
-        loadUsers();
-      }
     }
-  }, [loadMandates, loadOrganisations, loadUsers, mandates, organisations, user?.isAdmin, users]);
+  }, [loadOrganisations, organisations, user?.isAdmin]);
 
   const columns: GridColDef[] = [
     {
       field: "name",
       headerName: t("name"),
       type: "string",
-      editable: true,
       flex: 0.5,
       minWidth: 200,
     },
     {
       field: "mandates",
       headerName: t("mandates"),
-      editable: true,
       flex: 1,
       minWidth: 400,
-      type: "custom",
-      valueOptions: mandates,
-      getOptionLabel: (value: DataRow | string) => (value as Mandate).name,
-      getOptionValue: (value: DataRow | string) => (value as Mandate).id,
+      valueFormatter: (mandates: Mandate[]) => {
+        return mandates?.map(m => m.name).join(", ");
+      },
     },
     {
       field: "users",
       headerName: t("users"),
-      editable: true,
       flex: 1,
       minWidth: 400,
-      type: "custom",
-      valueOptions: users,
-      getOptionLabel: (value: DataRow | string) => (value as User).fullName,
-      getOptionValue: (value: DataRow | string) => (value as User).id,
+      valueFormatter: (users: User[]) => {
+        return users?.map(u => u.fullName).join(", ");
+      },
+    },
+    {
+      field: "actions",
+      type: "actions",
+      headerName: "",
+      flex: 0,
+      resizable: false,
+      cellClassName: "actions",
+      getActions: ({ id }) => [
+        <Tooltip title={t("edit")} key={`edit-${id}`}>
+          <GridActionsCellItem
+            icon={<EditOutlinedIcon />}
+            label={t("edit")}
+            onClick={() => startEditing(id)}
+            color="inherit"
+          />
+        </Tooltip>,
+      ],
     },
   ];
 
-  return isLoading ? (
-    <Stack sx={{ flex: "1 0 0", justifyContent: "center", alignItems: "center", height: "100%" }}>
-      <CircularProgress />
-    </Stack>
-  ) : (
-    <AdminGrid
-      addLabel="addOrganisation"
-      data={organisations as unknown as DataRow[]}
-      columns={columns}
-      onSave={onSave}
-      onDisconnect={onDisconnect}
-    />
+  return (
+    <>
+      <BaseButton
+        variant="outlined"
+        icon={<AddIcon />}
+        sx={{ marginBottom: "20px" }}
+        onClick={() => startEditing(0)}
```

Comment:
PP: Expliziteren funktionsalias neben startEditing definieren 
```suggestion
const startEditNew = () =>startEdit(0);

        onClick={() => startEditNew()}
```

Reply:
Ist m.M.n. unnötiger Code, da es bereits mit dem "Add"-Button klar ist, dass es sich um ein neues Objekt handelt, aber kanns im späteren PR zum Bereinigen schon ergänzen.

Reply:
War ein schwacher PP.